controllers/admin/AdminCartRulesController.php

```diff
@@ -530,6 +526,23 @@ public function getProductRuleDisplay($product_rule_group_id, $product_rule_id,
         return $this->createTemplate('product_rule.tpl')->fetch();
     }
 
+    public function populateCategories(&$flatCategories, $currentCategoryTree, $currentPath = ''): void
+    {
+        if (!$currentCategoryTree) {
+            return;
+        }
+
+        $separator = ' > ';
```

Comment:
Isn't it a configuration ?

Reply:
no, there is no need to change this I guess